Code/ThirdParty/tracy/LICENSE

```diff
@@ -0,0 +1,27 @@
+Tracy Profiler (https://github.com/wolfpld/tracy) is licensed under the
```

Comment:
Can you also add a "cgmanifest.json" file to specify which version and which commit hash of tracy this is exactly. For example look at Code/ThirdParty/Imgui. This makes it easier to update tracy in the future or investigate potential bugs etc.

Reply:
Added.